src/java/org/apache/cassandra/db/compaction/TimeWindowCompactionStrategyOptions.java

```diff
@@ -39,11 +39,11 @@
     protected static final int DEFAULT_EXPIRED_SSTABLE_CHECK_FREQUENCY_SECONDS = 60 * 10;
     protected static final Boolean DEFAULT_UNSAFE_AGGRESSIVE_SSTABLE_EXPIRATION = false;
 
-    protected static final String TIMESTAMP_RESOLUTION_KEY = "timestamp_resolution";
-    protected static final String COMPACTION_WINDOW_UNIT_KEY = "compaction_window_unit";
-    protected static final String COMPACTION_WINDOW_SIZE_KEY = "compaction_window_size";
-    protected static final String EXPIRED_SSTABLE_CHECK_FREQUENCY_SECONDS_KEY = "expired_sstable_check_frequency_seconds";
-    protected static final String UNSAFE_AGGRESSIVE_SSTABLE_EXPIRATION_KEY = "unsafe_aggressive_sstable_expiration";
+    public static final String TIMESTAMP_RESOLUTION_KEY = "timestamp_resolution";
```

Comment:
I am using these constants in `nodetool/stats`. IDEA even complains that having "protected static final" does not make sense when it is `final`. Instead of making public only the first two properties, I made them all public.